Makefile

```diff
@@ -330,14 +330,15 @@ DOCKER_GO = _() { $(SET_X); mkdir -p $(CURDIR)/.go/src/$${3:-dummy} ; mkdir -p $
 
 PARSE_PKGS=$(if $(strip $(EVE_HASH)),EVE_HASH=)$(EVE_HASH) DOCKER_ARCH_TAG=$(DOCKER_ARCH_TAG) KERNEL_TAG=$(KERNEL_TAG) PLATFORM=$(PLATFORM) ./tools/parse-pkgs.sh
 
+LINUXKIT_PKG_TARGET=build
+
 # buildkitd.toml configuration file can control a configuration of the buildkit daemon
 # it is used for example to setup a Docker registry mirror for CI to speedup the builds
 # this option can be overridden by setting BUILDKIT_CONFIG_FILE variable on make command line
 BUILDKIT_CONFIG_FILE ?= /etc/buildkit/buildkitd.toml
-BUILDKIT_CONFIG_OPTS := $(if $(wildcard $(BUILDKIT_CONFIG_FILE)),--builder-config $(BUILDKIT_CONFIG_FILE),)
+BUILDKIT_CONFIG_OPTS := $(if $(filter manifest,$(LINUXKIT_PKG_TARGET)),,$(if $(wildcard $(BUILDKIT_CONFIG_FILE)),--builder-config $(BUILDKIT_CONFIG_FILE),))
```

Comment:
I know this works, but it is ugly to read. And it depends on it _not_ containing `manifest`, when it probably should be checking that it _does_ contain `build`. Could we reverse it? Oh, but then there is `push`. Oy.

Reply:
Yes, I choose manifest because we need these options for both `build` and `push`...

Reply:
Unfortunately, I cannot come up with any better ideas.